fem/fespace.cpp

```diff
@@ -2637,6 +2643,39 @@ const Operator &L2ProjectionGridTransfer::BackwardOperator()
    return *B;
 }
 
+L2ElementRestriction::L2ElementRestriction(const FiniteElementSpace &fes)
+   : ne(fes.GetNE()),
+     vdim(fes.GetVDim()),
+     byvdim(fes.GetOrdering() == Ordering::byVDIM),
+     ndof(ne > 0 ? fes.GetFE(0)->GetDof() : 0)
+{ }
```

Comment:
Could we also add height and width of the operator? 
```
height = vdim*ne*ndof;
width = fes.GetVSize();
```
This is needed to set dimensions of localX and localY in bilinearform_ext.cpp, 
and for consistency with the other Element Restriction Operator, ndof -> ndofs.

Reply:
``height,width`` are members of the base class Operator.